yarn.lock

```diff
@@ -1238,6 +1238,13 @@ __metadata:
   languageName: node
   linkType: hard
 
```

Comment:
Please make sure to do a final `yarn install` whose results are included in the next updates -- `yarn.lock` changes for me when I check out your branch and run `yarn install` in it.

Also, I'd appreciate you rebasing this on top of `master`, because I'll eventually want that to have happened, and if it's not done already, there's minor risk something won't work when it's attempted in the Github landing process.